src/SamplesApp/SamplesApp.Skia.WPF/SamplesApp.Skia.WPF.csproj

```diff
@@ -1,6 +1,10 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 	<PropertyGroup>
-		<TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
+		<TargetFrameworks>net472;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
@jeromelaban, should we keep netcore 3.1?

Reply:
for WPF, the net5.0 tfm may not work as it may need the `-windows` suffix. The build will tell.

Reply:
Seems will need to switch to `-windows`